packages/calculator-bigint/package.json

```diff
@@ -36,7 +36,7 @@
     "build:clean": "rimraf ./{dist,temp}",
     "build:esm": "babel src --root-mode upward --extensions '.ts' --out-dir dist/esm --ignore '**/*/__tests__/'",
     "build:definitions": "tsc -b",
-    "build:types": "api-extractor run --local",
+    "build:types": "node ../../scripts/build-types.mjs",
```

Comment:
All the packages call the build-types script rather than `api-extractor` directly.